src/backend/electron/fileHelper.ts

```diff
@@ -9,10 +10,34 @@ export function writeFileSafely(
   path: string,
   data: string | NodeJS.ArrayBufferView,
 ) {
-  const tmpPath = `${path}.tmp`;
-  fs.writeFileSync(tmpPath, data);
+  let tmpPath: string;
+  let maxRetries = 16;
+  while (true) {
+    maxRetries--;
+    try {
+      // ランダムな文字列を8文字生成
+      const randStr = Math.floor(Math.random() * 36 ** 8)
+        .toString(36)
+        .padStart(8, "0");
+      tmpPath = `${path}-${randStr}.tmp`;
+      fs.writeFileSync(tmpPath, data, { flag: "wx" });
+      break;
+    } catch (error) {
+      const e = error as NodeJS.ErrnoException;
+      if (e.code !== "EEXIST" || maxRetries <= 0) {
+        throw e;
+      }
+    }
+  }
```

Comment:
結構ややこしさが上がってしまうので、重ならないランダムであるuuid4を使っちゃいますか！！
こうなるはず！
```suggestion
  const tmpPath = `${path}-${uuid4()}.tmp`;
  fs.writeFileSync(tmpPath, data, { flag: "wx" });
```
（８桁数字を提案したのは僕なのですみません 🙇 ）